isolated-buttons/src/app.js

```diff
@@ -0,0 +1,16 @@
+import {Subject} from 'rx'
+import {run} from '@cycle/core'
+import {makeDOMDriver} from 'cycle-snabbdom'
+
+import {createButton} from './Button'
+
+function main(sources) {
+  const Button = createButton({id: '0', removable: false})
+  const proxyAction$ = new Subject()
```

Comment:
This proxy is used just by the button, so why not put this proxy and the circular dep in the button component?